src/main/kotlin/net/atos/client/zgw/drc/util/Base64Converters.kt

```diff
@@ -0,0 +1,21 @@
+package net.atos.client.zgw.drc.util
```

Comment:
Minor thing but maybe we should move this out of the ZGW package altogether to some place more generic? Because I think we are now using it in the SmartDocuments service as well right? And it is very generic anyway. Maybe just move it to e.g. `nl.lifely.zac.util` (we should make a decision about these `atos` package names at some point..).